process/block/shardblock_test.go

```diff
@@ -3082,6 +3082,9 @@ func TestShardProcessor_CreateMiniBlocksShouldWorkWithIntraShardTxs(t *testing.T
 	}
 	shardCoordinator := mock.NewMultiShardsCoordinatorMock(3)
 	accntAdapter := &stateMock.AccountsStub{
+		RootHashCalled: func() ([]byte, error) {
+			return []byte("rootHash1"), nil
```

Comment:
I think this is not necessary for some tests. Maybe we should set `RootHashCalled` only where actually needed. Sorry if I'm mistaken.